controller/AdminAccessController.php

```diff
@@ -145,11 +150,38 @@ public function savePermissions()
         }
 
         return $this->returnJson([
-            'success'   => $success,
-            'message'   => $message
+            'success' => $success,
+            'message' => $message
         ], $code);
     }
 
+    /**
+     * Find users to assign access rights
+     */
+    public function findUser()
```

Comment:
what about tests?

Reply:
Do we really need to cover controller with unit tests?

Reply:
if we put everything into controller we do not need tests at all, right? :))

actually we do need test controllers. but currently it is a bit complex because of tao implementation.
so I'd extract sensitive parts into service and create tests for the service

Reply:
Already done in `UserService`